src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  width: $widthCard;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  background: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--margin-right-bottom {
+    margin-right: $marginRightCard;
+    margin-bottom: 46px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__objects {
+    padding: 0 16px;
+  }
+
+  &__image {
+    margin: 32px 19px 40px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    @extend %font-text;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %font-text;
+    margin-bottom: 16px;
+
+    font-weight: $font-weight-small;
+    font-size: $font-size-small;
+    line-height: $line-height-small;
+    color: $color-code-and-price;
+  }
+
+  &__display {
+    display: flex;
+    justify-content: space-between;
+
+    &--flex-b24 {
+      margin-bottom: 24px;
+    }
+
+    &--flex-b16 {
+      margin-bottom: 16px;
+    }
```

Comment:
same here, it is called atomic. You can add some atomic styles if you want, but this can't be inside bem block, you just create separate styles for that, e.g.
```
.mb-16 {
  margin-bottom: 16px
}
```
And just add this as a class to the element where you need it

Or find some lib which does that